internal/database/database_test.go

```diff
@@ -424,3 +427,90 @@ func testGetAccounts(t *testing.T, db DB) {
 		assert.Equal(t, 0, len(accounts))
 	})
 }
+
+func testModifiedTimeUpdate(t *testing.T, db DB) {
+	ctx := context.TODO()
```

Comment:
Since this test uses `time.Sleep` let's run it in parallel to avoid blocking other tests.

```suggestion
func testModifiedTimeUpdate(t *testing.T, db DB) {
        t.Parallel()
	ctx := context.TODO()
```